amazon_advertising_api/advertising_api.py

```diff
@@ -1151,9 +1214,12 @@ def _operation(self, interface, params=None, method='GET'):
 
         try:
             f = urllib.request.urlopen(req)
-            return {'success': True,
-                    'code': f.code,
-                    'response': f.read().decode('utf-8')}
+            return {
+                'success': True,
+                'api_version': self.api_version,
+                'code': f.code,
+                'data': f.read().decode('utf-8')}
```

Comment:
@DavidAntaramian Is it safe to rename 'response' -> 'data'? I would double check if nobody relies on this name. 'date' name came from the requirement 'https://pepsico-ecomm.atlassian.net/wiki/spaces/AUTO/pages/192643313/Automation+ETL+API+Requirements', if that's not critical we could keep 'response'

Reply:
@dkoryukova It's OK to rename here. The only other project I know that depends on this is sm-pipeline, and that is pinned against a specific commit, so it will stay stable.